format/Message.fbs

```diff
@@ -78,6 +214,34 @@ table DictionaryBatch {
   isDelta: bool = false;
 }
 
+table Xxhash64 {
+}
+
+table ExtensionDigest {
+  configuration : [KeyValue];
+}
+
+// Types of digests accepted.
+union DigestType  {
```

Comment:
"union" meaning we accept several digests for a single message?

Reply:
union meaning we might eventually want to allow different digests.  Only one for a single message.